graylog2-web-interface/src/components/common/Autocomplete/Autocomplete.test.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import { Formik } from 'formik';
+import { render, fireEvent, waitFor } from 'wrappedTestingLibrary';
+
+import Autocomplete from './Autocomplete';
+
+const OPTIONS = [
+  { value: 'Rojo', label: 'Red' },
+  { value: 'Verde', label: 'Green' },
+  { value: 'Amarillo', label: 'Yellow' },
+];
+
+describe('Autocomplete component', () => {
+  it('should render the field with a label', async () => {
+    const { getByText, baseElement } = render(
+      <Formik initialValues={{ value: 'Verde', label: 'Green' }} onSubmit={() => null}>
```

Comment:
We can also extract the form rendering to a function so that we can reuse it on the test below.